spec/system/provider_interface/provider_receives_email_when_a_new_application_has_been_submitted_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.feature 'A new application has been submitted' do
+  include CourseOptionHelpers
+
+  scenario 'the provider receives email', with_audited: true do
+    given_i_am_a_provider_user_with_a_provider
+    and_a_candidate_submited_their_application
+
+    when_the_application_is_delivered_to_my_provider
+
+    then_i_should_receive_an_email_with_a_link_to_the_application
+    and_an_audit_comment_has_submitted
+  end
+
+  def given_i_am_a_provider_user_with_a_provider
+    @course_option = course_option_for_provider_code(provider_code: 'ABC')
+    @provider_user = Provider.find_by(code: 'ABC').provider_users.first
+  end
+
+  def and_a_candidate_submited_their_application
+    @application_choice =
+      create(
+        :application_choice,
+        status: 'application_complete',
+        edit_by: Time.zone.today,
+        course_option: @course_option,
+        application_form:
+          create(
+            :completed_application_form,
+            submitted_at: Time.zone.today,
+),
```

Comment:
```suggestion
          ),
```

There's another small misalignment later in this file but it doesn't bug me